tests/ui/lint/lint-non-uppercase-usages.stderr

```diff
@@ -0,0 +1,64 @@
+warning: constant `my_static` should have an upper case name
+  --> $DIR/lint-non-uppercase-usages.rs:11:7
+   |
+LL | const my_static: u32 = 0;
+   |       ^^^^^^^^^
+   |
+   = note: `#[warn(non_upper_case_globals)]` on by default
+help: convert the identifier to upper case
+   |
+LL - const my_static: u32 = 0;
+LL + const MY_STATIC: u32 = 0;
+   |
+help: convert the identifier to upper case
+   |
+LL - const LOL: u32 = my_static + 0;
+LL + const LOL: u32 = MY_STATIC + 0;
+   |
+help: convert the identifier to upper case
+   |
+LL -     let _a = crate::my_static;
+LL +     let _a = crate::MY_STATIC;
```

Comment:
This seems pretty noisy. Could you convert this to a multipart suggestion?

Reply:
you could perhaps tweak the `help:` to say something like "convert the identifier *and its usages* to upper case"

Reply:
Sure, converted a multipart suggestion.